nservicebus/recoverability/exception-caveats.md

```diff
@@ -1,36 +1,34 @@
 ---
-title: Exception Caveats
-summary: Certain types of exceptions cannot be handled natively by NServiceBus.
+title: Exception caveats
+summary: NServiceBus cannot guarantee the handling of certain types of exceptions.
 component: core
-reviewed: 2018-09-21
+reviewed: 2018-12-28
 tags:
  - Exceptions
  - Error Handling
 redirects:
  - nservicebus/errors/exception-caveats
 ---
 
-Certain types of exceptions are behave differently and may require custom handling.
+NServiceBus cannot guarantee the handling of certain types of exceptions.
 
 
-## AccessViolationException
+## `AccessViolationException`
 
-If an [AccessViolationException](https://msdn.microsoft.com/en-us/library/system.accessviolationexception.aspx) is thrown, the endpoint will terminate. The reason is that a standard try-catch block, which NServiceBus uses, does not catch an `AccessViolationException`. As such it will bubble out of the handler and terminate the endpoint.
+If an [`AccessViolationException`](https://msdn.microsoft.com/en-us/library/system.accessviolationexception.aspx) is thrown, the endpoint is likely to terminate. This is because an `AccessViolationException` thrown by the common language runtime cannot be caught by a `try...catch` block.
 
-While these exceptions can be explicitly handled (using a [HandleProcessCorruptedStateExceptionsAttribute](https://msdn.microsoft.com/en-us/library/system.runtime.exceptionservices.handleprocesscorruptedstateexceptionsattribute.aspx)), Microsoft explicitly recommends not to do this.
+While this problem can be mitigated by using [`HandleProcessCorruptedStateExceptionsAttribute`](https://msdn.microsoft.com/en-us/library/system.runtime.exceptionservices.handleprocesscorruptedstateexceptionsattribute.aspx), Microsoft explicitly recommends not to do this.
 
 > Corrupted process state exceptions are exceptions that indicate that the state of a process has been corrupted.
 
-For more information see [Handling Corrupted State Exceptions](https://msdn.microsoft.com/en-us/magazine/dd419661.aspx#id0070035)
+For more information see [_Handling Corrupted State Exceptions_](https://msdn.microsoft.com/en-us/magazine/dd419661.aspx#id0070035).
 
 
-## StackOverflowException
+## `StackOverflowException`
 
-[StackOverflowExceptions](https://msdn.microsoft.com/en-us/library/system.stackoverflowexception.aspx) cannot be handled since .NET does not allow it.
+If a [`StackOverflowException`](https://msdn.microsoft.com/en-us/library/system.stackoverflowexception.aspx) is thrown, the process will terminate because the exception cannot be caught by a `try...catch` block.
```

Comment:
```suggestion
If a [`StackOverflowException`](https://docs.microsoft.com/en-us/dotnet/api/system.stackoverflowexception) is thrown, the process will terminate because the exception cannot be caught by a `try...catch` block.
```